WooCommerce/Classes/Analytics/AppStartupWaitingTimeTracker.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 import Yosemite
+import WooFoundation
```

Comment:
Just for consistency, we can change this one to:
```suggestion
import protocol WooFoundation.Analytics
```

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/12685/commits/725cf2a14bc1b3633b2a10caab220e28061dee49